public/app/plugins/datasource/loki/querybuilder/operations.ts

```diff
@@ -246,9 +246,10 @@ Example: \`\`error_level=\`level\` \`\`
       name: 'Line contains',
       params: [
         {
-          name: 'String',
+          name: '',
```

Comment:
With the previous version we would get the `String` appended to the `+` button which I kind of like:
![image](https://github.com/grafana/grafana/assets/8092184/3e0528e0-6a16-4479-a510-afaf06eb7c6f)

WDYT?

Reply:
Yeah, I removed it because I don't like it 😅 . Can add again if you find it helpful.

Reply:
In case we decide to go with an empty name, maybe we can trim the title here: https://github.com/grafana/grafana/blob/main/public/app/plugins/datasource/prometheus/querybuilder/shared/OperationParamEditor.tsx#L88

Reply:
Sure thing 
![imagen](https://github.com/grafana/grafana/assets/1069378/3eea24ef-ccf1-454f-8b68-bcb54cdead36)